test/album.js

```diff
@@ -76,4 +78,81 @@ describe('albums', () => {
         });
     });
   });
+
+  describe('/albums/:id POST', () => {
+    it('should order an albums without problems because are loged', done => {
```

Comment:
Some cases of error are missing

Reply:
as which?

is here. :P

Reply:
Missing album for example, try to buy multiples times same album, etc